ros_buildfarm/templates/devel/devel_task.Dockerfile.em

```diff
@@ -87,11 +104,6 @@ RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y ccache
     install_lists=install_lists,
 ))@
 
-@[if run_abichecker]@
-RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y abi-compliance-checker python3 python3-catkin-pkg-modules python3-pip 
```

Comment:
Why is `python3-pip` not needed anymore?

Reply:
it's installed here https://github.com/ros-infrastructure/ros_buildfarm/blob/20a4a62433d7806705e7184aef4520bd7caf0028/ros_buildfarm/templates/devel/devel_task.Dockerfile.em#L64

Reply:
> it's installed here

Only conditionally - see the line directly above:

https://github.com/ros-infrastructure/ros_buildfarm/blob/20a4a62433d7806705e7184aef4520bd7caf0028/ros_buildfarm/templates/devel/devel_task.Dockerfile.em#L63

Reply:
my bad, but I think we can remove `python3` because the command uses `python3` to launch the script.

Reply:
Restored https://github.com/ros-infrastructure/ros_buildfarm/pull/817/commits/469e8cad5ec41d0f9314d0713b45bf71d2162ae2